bug-bounty.md

```diff
@@ -0,0 +1,22 @@
+---
```

Comment:
Any reason this file is at the top-level of the project or am I reading things wrong?